common/pom.xml

```diff
@@ -23,12 +23,19 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
`web` should probably be in API only, since `common` does not deal with REST controllers.

Reply:
Before I put this in, it was failing on the JobServiceTest that expected a TransactionSystemException, perhaps it's a transitive dependency of web, or something with how web integrates with JPA for handling some of the @Pattern validation?

Reply:
Actually doing just this works:
<dependency>
      <groupId>org.springframework.boot</groupId>
      <artifactId>spring-boot-starter-validation</artifactId>
      <version>2.2.0.RELEASE</version>
      <scope>compile</scope>
      <exclusions>
        <exclusion>
          <artifactId>tomcat-embed-el</artifactId>
          <groupId>org.apache.tomcat.embed</groupId>
        </exclusion>
      </exclusions>
    </dependency>

Reply:
Github removed the brakcets of the XML from above.